webservices/resources/candidate_aggregates.py

```diff
@@ -192,3 +202,24 @@ def build_query(self, **kwargs):
         query = filters.filter_range(query, kwargs, self.filter_range_fields(models.CandidateTotal))
         query = filters.filter_fulltext(query, kwargs, self.filter_fulltext_fields)
         return query
+
+    """
+        Helper method that prints out what the actual SQL query is, I had problems figuring out
```

Comment:
Were you saying this moring that this is not better than just doing a print? Should we keep this @jontours